NEWS.md

```diff
@@ -1,5 +1,20 @@
 # mkosi Changelog
 
+## v17
+
+- Added ToolsTreePackages= to add extra packages to the default tools
+  tree.
+- Removed dnf from Debian default tools tree packages as it was removed
```

Comment:
uh, what? hang on a sec and let me see what the heck is going on there

Reply:
It just didn't migrate to testing because of some regression, but this did make trixie tools trees fail to build.

Reply:
worse than that, it was removed, because it fails to build and nobody fixed it, let me throw something at it

Reply:
But not point really to wait with the release for dnf to be fixed in Debian. dnf missing was the reason to add `ToolsTreePackages=` to make it easy to add it (or anything else for that matter) back while keeping the comfort of just writing `default` for the tools tree setting instead of writing a new preset file.

Reply:
If it's easy to fix I'd prefer to add back dnf to the default list of packages to be fair

Reply:
I've uploaded a workaround, but it will take a week or so to reach testing

Reply:
Let's add back dnf then, it means debian default tool trees will be broken for a week, but that shouldn't be a huge issue.